lib/pbench/cli/agent/commands/tools/clear.py

```diff
@@ -36,12 +34,15 @@ def execute(self):
             # We were not given any remotes on the command line, build the list
             # from the tools group directory.
             remotes = self.remote(self.tool_group_dir)
+            if not remotes:
+                self.logger.error(f'No such group "{self.context.group}".')
+                return 1
 
         for remote in remotes:
```

Comment:
I guess we also need to do 
```
groups = self.context.group.split(",")
for group in groups:
```
to support multiple comma separated groups

Reply:
Maybe in the next PR.  😉